src/components/crown/utils.js

```diff
@@ -0,0 +1,10 @@
+export function removeFlows(graph, shape) {
+  graph.getConnectedLinks(this.shape).forEach(shape => this.$emit('remove-node', shape.component.node));
+  shape.getEmbeddedCells({ deep: true }).forEach(cell => {
+    if (cell.component) {
+      graph.getConnectedLinks(cell).forEach(shape => this.$emit('remove-node', shape.component.node));
+      shape.unembed(cell);
+      this.$emit('remove-node', cell.component.node);
+    }
+  });
+}
```

Comment:
I think this could probably be rewritten as a single pipeline, which would remove some of the nesting and make this more readable IMO. Something like this:
`shape.getEmbeddedCells({ deep: true })
    .filter(cell => cell.component)
    .flatMap(cell => graph.getConnectedLinks(cell))
    .foreach(shape => {
        this.$emit('remove-node', shape.component.node));
        shape.unembed(cell);
        this.$emit('remove-node', cell.component.node);
    });
`